### DISCUSSION TITLE
Schéma causal

### DISCUSSION THREAD

**Discussion by Marine1719** [BIO6032/2022-SEIR-reproduction#29] [Ideas]
Salut à tous, 
Je voulais juste avoir votre avis pour le schéma causal que je suis en train de faire. Est-ce que pour voir ce serait plus visuel d'ajouter des couleurs aux flêches? Je pensais peut-être mettre les flêches qui sont associés au même taux en couleurs pour que ce soit plus clair, je ne sais pas si ça vous semble judicieux ou si je laisse tout en noir ? Merci

**Comment by salomebsk** [comment]
Salut!
Pour le schéma causal, je pense qu'il serait mieux de garder les flèches en noir sinon on risque d'avoir trop de couleur ce qiu pourrait être plus mélangeant qu'autre chose ? Le seul taux qui est présent à différents endroits est le recovery rate fr. Éventuellement celui-ci pourrait avoir des flèches un peu différentes des autres taux (plus fines ou en pointillés par exemple).. un peu comme dans l'article mais différement (la manière dont ils ont présenté les fléches n'était pas super clair au premier abord je trouve). Vous en dites quoi ?

**Reply by salomebsk** [reply]
Éventuellement on pourrait aussi faire la même chose pour le taux d'infection puisqu'il est dépendant de différentes catégories (susceptibles, asymptomatiques, symptomatiques) puis expliquer ça dans la légende.

**Comment by Marine1719** [comment]
J'ai vais mettre un premier exemple, dites moi ce que vous en pensez, je changerais selon les commentaires :)

**Reply by salomebsk** [reply]
Le schéma est très beau, aéré et clair! 'Susceptible outside campus' devrait être Exogeneous input... ce sont des personnes en dehors de l'université qui peuvent infecter des personnes de l'université, et pas des susceptibles.
En fait les Susceptibles sont ceux qui peuvent devenir infectés par un membre de l'uni (asymptomatique A ou symptomatique I) ou quelqu'un de l'extérieur de l'uni (E), d'après ce que j'ai compris..  Il faudrait peut-être modifier cette case :)

**Reply by salomebsk** [reply]
Aussi, on pourrait ajouter les noms utilisés pour chaque variable pour mieux les identifier dans les équations (genre A, I, S...)

**Comment by salomebsk** [comment]
Tu en dis quoi ?

**Reply by Marine1719** [reply]
J'ai changé :)
